src/test/java/stringCalculator/Calculator.java

```diff
@@ -0,0 +1,95 @@
+package stringCalculator;
+
+import java.util.Arrays;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.*;
+
+public class Calculator {
+
+    final String word;
+    final String[] delimiters;
+    final CustomDelimiterCondition customCondition;
+
+    public Calculator(String word, String[] delimiters, CustomDelimiterCondition customCondition){
+       this.word = word;
+       this.delimiters = delimiters;
+       this.customCondition = customCondition;
+    }
```

Comment:
여긴 space 3칸을 사용하셨는데 4칸으로 맞춰주시면 좋을 것 같아요. 전체적으로 컨벤션은 신경 써주시면 감사합니다. 
```suggestion
    public Calculator(String word, String[] delimiters, CustomDelimiterCondition customCondition) {
        this.word = word;
        this.delimiters = delimiters;
        this.customCondition = customCondition;
    }
```

Reply:
앗 넵